modules/dcache/src/main/java/diskCacheV111/poolManager/RequestContainerV5.java

```diff
@@ -1396,7 +1407,7 @@ private void stateEngine( Object inputObject ) {
                         return ;
                     }
 
-                    if( ( rc = askIfAvailable() ) == RT_FOUND ){
+                    if( ( rc = askIfAvailable() ) == ReturnCode.FOUND.value ){
```

Comment:
Thanks for your explaination, i'll change the code. Which name do you suggest for the enum ? : D